### DISCUSSION TITLE
Weather Conditions

### DISCUSSION THREAD

**Discussion by genitr** [AscensionGameDev/Intersect-Engine#1520] [Ideas]
**Is your feature request related to a problem? Please describe.**
The weather in the game is essentially a static function, because it cannot be changed in the future.  After all, if you initially determined the weather for the map, then it will never change.

**Describe the solution you'd like**
I suggest adding a weather display condition to the map settings.  It is even possible to make several weather effects and, depending on the conditions, have different weather.

**Additional context**

**Comment by WeylonSantana** [comment]
![image](https://user-images.githubusercontent.com/63019821/188620892-3a416ca8-4712-4184-a88c-41c3cb669961.png)
